x/evm/genesis.go

```diff
@@ -20,9 +20,9 @@ func InitGenesis(
 	ctx sdk.Context,
 	k keeper.Keeper,
 	accountKeeper types.AccountKeeper, // nolint: interfacer
+	bankKeeper types.BankKeeper,
 	data GenesisState,
```

Comment:
is there a reason why we use `GenesisState` here and not `types.GenesisState` like in exportgenesis?